use-product-search.md

```diff
@@ -0,0 +1,304 @@
+---
+title: Using the Product Search
+excerpt: It is possible to find products by various criteria using the `ProductSearch` class, including name, slug, taxons, channels, price ranges, sorting, and custom attributes.
+date: 2025-02-17
+tags: [models, product]
+---
+
+The `ProductSearch` class provides a powerful and flexible way to retrieve products based on various criteria. 
+
+It supports:
+
+> - Filtering by name, slug, taxon, price range, and property values
+> - Sorting options and pagination
+> - Custom attribute filtering
```

Comment:
The intro text and the items belong together, whereas the blockquote marks a highlighted separate block in the text, therefore it needs to be removed.

```suggestion
It supports:

- Filtering by name, slug, taxon, price range, and property values
- Sorting options and pagination
- Custom attribute filtering
```

Mind that such elements in the texts are for semantic structuring and not for decoration purposes.